specification/keyvault/resource-manager/Microsoft.KeyVault/preview/2021-06-01-preview/keys.json

```diff
@@ -633,8 +633,7 @@
         },
         "data": {
           "description": "Blob encoding the policy rules under which the key can be released.",
-          "type": "string",
-          "format": "base64url"
+          "type": "string"
```

Comment:
Why should this differ from the data plane? That may be a jarring experience for end users.

See https://github.com/Azure/azure-rest-api-specs/blob/2b083f0d1b7ecb0717741551fa543a73e86d672c/specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.3-preview/keys.json#L1304-L1309

Reply:
If the goal was to make https://docs.microsoft.com/en-us/azure/templates/microsoft.keyvault/vaults/keys?tabs=json correct, this change still won't do it. Even an empty JSON object like `{}` is not a string type, nor is it `null`. The swagger should be kept as-is and the doc should be corrected. Note that it is a preview doc so changes like that - while perhaps not ideal - should be expected.

That said, I'm not entirely sure where that example in the aforementioned doc comes from. It doesn't seem to correspond to any example data under the "examples/" subdirectory, nor is there an *edit* link to edit the doc and find the path to it. You might ask @jlichwa or the original author of the page.